packages/select/src/FilterMultiSelect/provider/SelectionProvider/SelectionProvider.tsx

```diff
@@ -34,19 +34,15 @@ export const SelectionProvider = (
   props: SelectionProviderProps
 ): JSX.Element => {
   const { onSearchInputChange, ...otherProps } = props
-  const isFirstRender = React.useRef(true)
   const [searchQuery, setSearchQuery] = useState<string>("")
 
   /**
    * While the onSearchInputChange  is a side-effect, this useEffect should be fine.
    * If the search input state becomes controlled, this useEffect could cause synchronisation
    * issues and should be replaced.
    */
+
   React.useEffect(() => {
-    if (isFirstRender.current) {
-      isFirstRender.current = false
```

Comment:
Allow `onSearchInputChange(searchQuery)` to be called on first render as the `searchQuery` will be ""